### DISCUSSION TITLE
🎅 Santa is coming - Day 15!

### DISCUSSION THREAD

**Discussion by CarolineMorton** [CarolineMorton/AdventOfCode_WIR#18] [Ideas]
Wow Day 15 is long! 

We have Lanternfish and Submarines!

**Comment by AR2202** [comment]
I've done [Day15 in Haskell](https://github.com/AR2202/AOC2024/blob/main/src/Day15.hs), but returned to Rust for some of the later ones. Anyone still interested in discussing those?
